doc/api/cluster.md

```diff
@@ -807,9 +812,9 @@ before last `'disconnect'` or `'exit'` event is emitted.
 ```js
 // Go through all workers
 function eachWorker(callback) {
-  for (var id in cluster.workers) {
+  Object.keys(cluster.workers).forEach((id) => {
```

Comment:
Should I also change [the prototype](https://github.com/nodejs/node/blame/master/doc/api/cluster.md#L223) into the `for`...`in` variant?